examples/Building-and-deploying-multi-stage-RecSys/02-Deploying-multi-stage-RecSys-with-Merlin-Systems.ipynb

```diff
@@ -75,20 +75,53 @@
     "\n",
     "In case you need to install them for running this example on GPU, execute the following script in a cell.\n",
     "```\n",
-    "%pip install \"feast<0.20\" faiss-gpu\n",
+    "%pip install feast faiss-gpu\n",
```

Comment:
This should be anchored to 0.31. Given this is not a 1.0+ version the api may change again.

Reply:
thanks for the feedback. let me change that.